Dub/Dub.Web.Core/ApiStatusCode.cs

```diff
@@ -70,5 +70,59 @@ public enum ApiStatusCode
         /// Login for this account is disallowed.
         /// </summary>
         LoginNotAllowedError = RangeAccount + 7,
+        
+        /// <summary>
+        /// Indicating user lockout is not enabled.
+        /// </summary>
+        UserLockoutNotEnabled = RangeAccount + 8,
+        
+        /// <summary>
+        /// Indicating a user already has a password.
+        /// </summary>
+        UserAlreadyHasPassword = RangeAccount + 9,
+        
+        /// <summary>
+        /// Idicating a concurrency failure.
+        /// </summary>
+        ConcurrencyFailure = RangeAccount + 10,
+        
+        /// <summary>
+        /// Idicating an invalid token.
+        /// </summary>
+        InvalidToken = RangeAccount + 11,
+        
+        /// <summary>
+        /// Idicating a password mismatch.
+        /// </summary>
+        PasswordMismatch = RangeAccount + 12,
```

Comment:
Rename to `IncorrectPassword` that one of common names for such errors, would be easier to understand later on what is that.